docs/DEPLOYMENT.md

```
@@ -0,0 +1,78 @@
+| Pod | Purpose |
+|-----|---------|
+| `control-plane` | Fastify API server (port 4000) |
+| `dashboard` | Next.js frontend (port 3000) |
+| `postgresql-{1,2,3}` | CloudNativePG 3-node cluster |
+| `postgresql-rw-pooler` (x2) | PgBouncer connection pooling |
+| `qdrant` | Vector database |
+| `tailscale-proxy` | Ingress via Tailscale |
+
+## CI/CD Pipeline
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)